react/components/EXPERIMENTAL_Charts/ScatterChart/README.md

```diff
@@ -0,0 +1,23 @@
+#### The Scatter charts shows the data as a collection of points, they useful for detect if a relationship or correlation of multiple variables. 
```

Comment:
@fanny did you apply the suggestion? I still see `they useful for detect` here.

Reply:
Strange, but done!